monkey/monkey_island/cc/server_utils/consts.py

```diff
@@ -37,9 +37,7 @@ def _get_monkey_island_abs_path() -> str:
     _MONGO_EXECUTABLE_PATH_WIN if is_windows_os() else _MONGO_EXECUTABLE_PATH_LINUX
 )
 
-DEFAULT_SERVER_CONFIG_PATH = expand_path(
-    os.path.join(MONKEY_ISLAND_ABS_PATH, "cc", SERVER_CONFIG_FILENAME)
-)
+DEFAULT_SERVER_CONFIG_PATH = str(Path(MONKEY_ISLAND_ABS_PATH, "cc", SERVER_CONFIG_FILENAME))
```

Comment:
Why convert this back to string?

Reply:
1. I'm trying to limit the scope of this refactor. `DEFAULT_SERVER_CONFIG_PATH` was a string before, and now it still is.
2. This is an established pattern within this file: https://github.com/guardicore/monkey/blob/9ed21458107a4f4a9329cf703ff0eeb8cfe7ef1c/monkey/monkey_island/cc/server_utils/consts.py#L48-L49

Changing these constants from strings to Path objects should be done in a different PR.